src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonElementParseTests.cs

```diff
@@ -49,6 +49,27 @@ public static void TryParseValue(string json, JsonValueKind kind)
             Assert.False(element!.Value.SniffDocument().IsDisposable());
         }
 
+        [Fact]
+        public static void ParseValue_AllowMultipleValues_TrailingJson()
+        {
+            var options = new JsonReaderOptions { AllowMultipleValues = true };
+            var reader = new Utf8JsonReader("[null,false,42,{},[1]]             null"u8, options);
+
+            JsonElement element = JsonElement.ParseValue(ref reader);
+            Assert.Equal("[null,false,42,{},[1]]", element.GetRawText());
```

Comment:
If someone wanted to access all the top-level values (e.g., in this case, wanted to be able to read the trailing `null` as well), is your recommendation something like this?

```
// Pseudocode - didn't check this compiles
Span<byte> sourceData = "[null,false,42,{},[1]]             null"u8;
while (sourceData.Length > 0)
{
    var reader = new Utf8JsonReader("[null,false,42,{},[1]]             null"u8, options);
    JsonElement element = JsonElement.ParseValue(ref reader);
    // Do something with element
    sourceData = sourceData.Slice(element.GetRawText().Length);
}
```

... or is there some way to continue getting more top-level values out of the existing `Utf8JsonReader`?

Reply:
It's possible to chain multiple `ParseValue` calls on the same `Utf8JsonReader` instance. I've updated the test to better illustrate this.

Reply:
Your example should still work though, although it does require a bit more ceremony compared to reusing the same reader.